### PR TITLE
Ch setup database 159316017

### PR DESCRIPTION
**Pull request by koiic**
#### Relevant pivotal tracker story
 [#159316017](https://www.pivotaltracker.com/n/projects/2183409/stories/159316017)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (fidelisojeah) [inline .env]: I do not want to see this file

File: server/model/dropTable.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import db from '../config/connect';
```
Comment (fidelisojeah) [inline server/model/dropTable.js]: Move this file to `migrations/dropTable.js`

File: server/model/setupTables.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import db from '../config/connect';
```
Comment (fidelisojeah) [inline server/model/setupTables.js]: Move this file to `migrations/setupTable.js`

File: server/model/setupTables.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+db.query('CREATE TABLE users(id SERIAL PRIMARY KEY, email VARCHAR(255) UNIQUE NOT NULL, firstname VARCHAR(255), lastname VARCHAR(255))');
```
Comment (fidelisojeah) [inline server/model/setupTables.js]: Also, this can be more readable

File: UI/signin.html [hunk 5]
```diff
@@ -36,34 +36,38 @@
...
-                    <div class="column">
-                        <div class="form">
-                            <h1 class="text-center nav__logo app-color">Welcome Back</h1>
-                            <div class="form-card">
-                                <form>
-                                    <div class="form-row">
-                                        <label for="name">Email:</label>
-                                        <input type="email" class="form-control" placeholder="Enter your email" required minlength="3" maxlength="30">
-                                    </div>
+            <div class="shelter">
```
Comment (fidelisojeah) [inline UI/signin.html:39]: Do you not believe this should be in another PR/branch?

**Review by fidelisojeah — CHANGES_REQUESTED** [review]

File: UI/signin.html (hunk 5 above)
Comment (koiic) [reply UI/signin.html:39]: alright,i'll work on it

File: server/model/setupTables.js (hunk 4 above)
Comment (koiic) [reply server/model/setupTables.js]: Okay.

File: .env (hunk 1 above)
Comment (koiic) [reply .env]: Okay,I thought i already removed it

Comment (koiic) [reply .env]: I will remove it now
